src/Core/Content/Newsletter/SalesChannel/NewsletterSubscribeRoute.php

```diff
@@ -274,6 +276,9 @@ private function getOptInValidator(SalesChannelContext $context, bool $validateS
                 ->add('storefrontUrl', new NotBlank(), new Choice(array_values($this->getDomainUrls($context))));
         }
 
+        $validationEvent = new BuildValidationEvent($definition, new DataBag(), $context->getContext());
```

Comment:
Pass the data bag from the called method to this method and use it here

Reply:
Thank you @shyim,

you're absolutly right. 🤗